src/Admin/AdminConsole/Models/OrganizationEditModel.cs

```diff
@@ -179,13 +179,29 @@ public OrganizationEditModel(Provider provider)
      * This is mapped manually below to provide some type safety in case the plan objects change
```

Comment:
💭 Does the comment here still hold true even with the changes to how the helper functions?

Reply:
@cturnbull-bitwarden - non-blocking comment above, but still want to address, if possible!

Reply:
You're absolutely right to bring this up! My intention wasn't to break the spirit of the comment. As it is, this is no longer safe from rename refactors, so I'll go ahead and update this to not use anonymous type properties.